### DISCUSSION TITLE
User Interface Issue on latest version(s) of Glances

### DISCUSSION THREAD

**Discussion by bluecasita** [nicolargo/glances#3002] [General]
See discussion here:  https://github.com/nicolargo/glances/discussions/2986

Some of us are having strange display issues on the most recent releases of Glances (4.20 and 4.2.1).  4.1.1 works flawlessly.  This does not appear to be limited to a specific browser or OS.

Have changes been made to Glances to cause this or is it a weird anomalie?

**Comment by nicolargo** [comment]
You should upgrade Glances to 4.2.1 and do not forget to clear your Web browser cache (F5 or CTRL-F5).

**Reply by bluecasita** [reply]
I did that... My background colors all went back to normal but for some reason the text is still wrapping in the "Command" column causing the row height to be doubled on several lines. I prefer having to scroll to the right and having everything on one line so it looks consistent. 

If you think this issue is also fixed, I can try again, but I did test multiple browsers and cleared my cache each time. 

Just to confirm, the :latest tag does point to 4.2.1, correct?
